Sources/Datadog/Tracing/DDSpanContext.swift

```diff
@@ -16,6 +18,21 @@ internal struct DDSpanContext: OTSpanContext {
     /// The baggage items of this span.
     let baggageItems: BaggageItems
 
+    let activityId: os_activity_id_t
+    var activityState = os_activity_scope_state_s()
```

Comment:
are we going to use them in the future? or can we make them private?

Reply:
I have refactored the code and moved this logic to DDSpan, and have made it private there. Also all the logic interacting with ActiveSpansPool is now done in DDSPan, so the variables can be kept private